diff -Naur udisks-2.1.3.orig/data/org.freedesktop.udisks2.policy.in udisks-2.1.3/data/org.freedesktop.udisks2.policy.in
--- udisks-2.1.3.orig/data/org.freedesktop.udisks2.policy.in	2014-03-05 23:05:43.000000000 +0700
+++ udisks-2.1.3/data/org.freedesktop.udisks2.policy.in	2014-11-12 22:46:26.834074400 +0700
@@ -29,7 +29,7 @@
     <defaults>
       <allow_any>auth_admin</allow_any>
       <allow_inactive>auth_admin</allow_inactive>
-      <allow_active>auth_admin_keep</allow_active>
+      <allow_active>yes</allow_active>
     </defaults>
   </action>
 
